pkg/prompts/prompts.go

```diff
@@ -79,6 +82,21 @@ func RunPromptsFromConfigWithSkipsIO(config *config.DraftConfig, varsToSkip []st
 // GetVariableDefaultValue returns the default value for a variable, if one is set in variableDefaults from a ReferenceVar or literal VariableDefault.Value in that order.
 func GetVariableDefaultValue(variableName string, variableDefaults []config.BuilderVarDefault, inputs map[string]string) string {
 	defaultValue := ""
+
+	if variableName == "APPNAME" {
+		dirName, err := getCurrentDirName() //draft
+		if err != nil {
+			log.Errorf("Error retrieving current directory name: %v", err)
```

Comment:
does %v work? typically it's only for formatting errors

Reply:
it will sanitize %v to v here - defaultValue = sanitizeAppName(dirName)

Reply:
ah. for some reason I misread it as %w.

I'd recommend switching to %s so we can a human readable error output for this.